Makefile

```diff
@@ -92,6 +92,35 @@ make test-integration:
 	make -C ./tests/integration test
 
 
+# Changelog management
+## We use Towncrier (https://towncrier.readthedocs.io) for changelog management
+
+ENTRY_TEXT ?= 'Use [Semantic Versioning](https://semver.org/spec/v2.0.0.html) for your changelog entry'
+CHANGE_ID ?= 9999
+CHANGE_TYPE ?= changed
```

Comment:
What are other options? Can we validate correctness of the type?

Reply:
The types are: `breaking`, `added`, `changed` and `fixed`. They're listed in `towncrier.toml`, and we do validate correctness.